modules/reindex/src/test/java/org/elasticsearch/reindex/ReindexBasicTests.java

```diff
@@ -37,21 +41,32 @@ public void testFiltering() throws Exception {
         );
         assertHitCount(prepareSearch("source").setSize(0), 4);
 
+        TestTelemetryPlugin testTelemetryPlugin = getTestTelemetryPlugin();
```

Comment:
I think that we should write a new test case for the newly introduced metrics instead of updating an existing test

Reply:
yes, it looks like it's an integration test. Integration tests should go into the `internalClusterTest` folder.

Reply:
I think we can move the `ReindexBasicTests` into `internalClusterTest` and write a new test case, wdyt?

Reply:
> I think we can move the ReindexBasicTests into internalClusterTest and write a new test case, wdyt?

I've just noticed that you wrote a new test, we can do what I suggested here in a follow-up PR.